app/controllers/events/view/scheduler.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  header: {
+    left   : 'today prev,next',
+    center : 'title',
+    right  : 'timelineDay,timelineThreeDays,agendaWeek,month'
+  },
+  views: {
+    timelineThreeDays: {
+      type     : 'timeline',
+      duration : { days: 3 }
+    }
+  },
+  actions: {
+    drop() {
+      console.log('drop');
+    },
+    eventReceive(event) {
```

Comment:
@niranjan94 These default callbacks are not working when I use `draggable=true` Can you please point me in the right direction.
The things I tried : 
 - using `ondragstart`, `ondragover` and `ondrop` default attributes of html 5 to call actions. 
But I am not sure on how to make the scheduler respond to them.
- just testing out by pasting entire jquery-ui in the console  to make sure my callbacks are working otherwise , they did.